### PR TITLE
chuck: Add version 1.4.0.0

### PR DESCRIPTION
**Pull request by ddavness**
ChucK is a strongly-timed, music oriented programming language.

**Considerations:**
- The installer comes with a GUI tool (`miniAudicle.exe`). A shortcut was not added, although it might be a good idea to do so given it's usefulness (but clearly not required to be able to work with it!);
- At the moment, ChucK is distributed by two universities: [Stanford](http://chuck.stanford.edu) and [Princeton](https://chuck.cs.princeton.edu). While the former is apparently the "official" one, the latter is (from my testing) faster + supports https.

### CODE REVIEW

File: bucket/chuck.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.4.0.0",
+    "description": "Strongly-timed, Concurrent, and On-the-fly Music Programming Language.",
+    "homepage": "http://chuck.stanford.edu",
+    "license": "GPL-2.0-only",
+    "url": "http://chuck.stanford.edu/release/files/exe/chuck-1.4.0.0.msi",
+    "hash": "d3f24d49222dd18d0c7e3b3bb2d2965ad68dccd8e774723c62ad7eb33127a6d5",
+    "extract_dir": "PFiles/ChucK",
```
Comment (linsui) [inline bucket/chuck.json]:
```suggestion
    "extract_dir": "PFiles\\ChucK",
```

File: bucket/chuck.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "bin": [
+        "bin/chuck.exe",
```
Comment (linsui) [inline bucket/chuck.json]:
```suggestion
        "bin\\chuck.exe",
```

File: bucket/chuck.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "license": "GPL-2.0-only",
+    "url": "http://chuck.stanford.edu/release/files/exe/chuck-1.4.0.0.msi",
+    "hash": "d3f24d49222dd18d0c7e3b3bb2d2965ad68dccd8e774723c62ad7eb33127a6d5",
+    "extract_dir": "PFiles\\ChucK",
+    "bin": [
+        "bin\\chuck.exe",
+        "miniAudicle.exe"
+    ],
+    "checkver": {
+        "url": "http://chuck.stanford.edu/release/",
```
Comment (Ash258) [inline bucket/chuck.json]: Get version from <http://chuck.stanford.edu/release/VERSIONS> with `^([\d.+])$`

Comment (ddavness) [reply bucket/chuck.json]: Using `^([\d.]+)` instead.

**Pull request merged**